docs/semgrep-ci/configuration-reference.md

```diff
@@ -10,7 +10,7 @@ import MoreHelp from "/src/components/MoreHelp"
 Configure Semgrep CI by passing these environment variables in your CI job.
 
 :::info
-While environment variables are the preferred way to configure Semgrep CI, pass any of these options as command-line options. Refer to the output of `semgrep-agent --help` to find the corresponding flags.
+While environment variables are the preferred way to configure Semgrep CI, pass any of these options as command-line options. Refer to the output of TODO `semgrep-agent --help` to find the corresponding flags.
```

Comment:
Should we point to any specific help of semgrep ci, or just substitute this with semgrep --help? Not sure if there will be the information user searches for.

Reply:
We can reference `semgrep ci --help` which is specific to that subcommand, but I think it has less info that semgrep-agent's used to have. @brendongo did we make sure to port the same env variables we used before in `GitMeta`? I think we need to mention them in the `--help` text.

Reply:
@brendongo please check the [comment above](https://github.com/returntocorp/semgrep-docs/pull/472#discussion_r839083736).

Reply:
I think since we're moving away from using a github action and instead telling people to run `semgrep ci` command directly they can use command line arguments instead of flags. I think we can delete this section `# CI configuration reference` as unnecessary

Reply:
They are not anymore. So with the new `semgrep ci` people pass in relevant arguments as flags `semgrep ci --timeout 30` the env vars are generally not available anymore

Reply:
Clarified in DMs, env vars still available for Semgrep CI.